templates/company/contracts/CompanyTemplate.sol

```diff
@@ -30,48 +24,66 @@ contract CompanyTemplate is BaseTemplate {
         _ensureMiniMeFactoryIsValid(_miniMeFactory);
     }
 
-    function newTokenAndInstance(string _id, address[] _holders, uint256[] _stakes) public {
-        newToken();
-        newInstance(_id, _holders, _stakes);
+    function newTokenAndInstance(
+        string _id, 
+        address[] _holders, 
+        uint256[] _stakes, 
+        string _tokenName, 
+        string _tokenSymbol,
```

Comment:
Super small thing, but the parameter order _may_ be a little bit more obvious if we had the arguments in the same order they would be used in, i.e. `_tokenName, `_tokenSymbol`, `_id`, `_holders`, ...

That way a frontend could _conceivably_ do something like `creationArgs = [].concat(tokenArgs, instanceArgs)` and it'd just work (not that we would likely be this loose in our own frontend).